subt_ign/worlds/finals_systems_prelim2.dot

```diff
@@ -0,0 +1,125 @@
+  18  [label="18::megacavern_tile_18::tile_18"];
+  19  [label="19::megacavern_tile_19::tile_19"];
+  20  [label="20::megacavern_tile_20::tile_20"];
+  21  [label="21::megacavern_tile_21::tile_21"];
+  22  [label="22::megacavern_tile_22::tile_22"];
+  23  [label="23::megacavern_tile_23::tile_23"];
+  24  [label="24::megacavern_tile_24::tile_24"];
+  25  [label="25::megacavern_tile_25::tile_25"];
+  26  [label="26::megacavern_tile_26::tile_26"];
+  27  [label="27::megacavern_tile_27::tile_27"];
+  28  [label="28::megacavern_tile_28::tile_28"];
+  29  [label="29::megacavern_tile_29::tile_29"];
+  30  [label="30::megacavern_tile_30::tile_30"];
+  31  [label="31::megacavern_tile_31::tile_31"];
+  32  [label="32::megacavern_tile_32::tile_32"];
+  33  [label="33::megacavern_tile_33::tile_33"];
+  34  [label="34::megacavern_tile_34::tile_34"];
+  35  [label="35::megacavern_tile_35::tile_35"];
+  36  [label="36::megacavern_tile_36::tile_36"];
+  37  [label="37::megacavern_tile_37::tile_37"];
+  38  [label="38::megacavern_tile_38::tile_38"];
+  39  [label="39::megacavern_tile_39::tile_39"];
+  40  [label="40::megacavern_tile_40::tile_40"];
+  41  [label="41::megacavern_tile_41::tile_41"];
+  42  [label="42::megacavern_tile_42::tile_42"];
+  43  [label="43::megacavern_tile_43::tile_43"];
+  44  [label="44::megacavern_tile_44::tile_44"];
+  45  [label="45::megacavern_tile_45::tile_45"];
+  46  [label="46::megacavern_tile_46::tile_46"];
+  47  [label="47::megacavern_tile_47::tile_47"];
+  48  [label="48::megacavern_tile_48::tile_48"];
+
+  /* ==== Edges ==== */
+
+  0  -- 1   [label=1];
+  1  -- 2   [label=1];
+  2  -- 3   [label=1];
+  2  -- 14  [label=3];
+  2  -- 25  [label=1];
+  3  -- 4   [label=3];
+  4  -- 5   [label=6];
+  4  -- 48  [label=6];
+  5  -- 6   [label=3];
+  5  -- 15  [label=3];
+  6  -- 7   [label=3];
+  6  -- 8   [label=3];
+  6  -- 19  [label=3];
+  7  -- 8   [label=3];
+  8  -- 11  [label=3];
+  9  -- 10  [label=3];
+  9  -- 11  [label=3];
+  10 -- 42  [label=6];
+  11 -- 13  [label=3];
+  11 -- 42  [label=6];
+  12 -- 43  [label=3];
+  12 -- 45  [label=3];
+  13 -- 40  [label=6];
+  13 -- 41  [label=3];
+  14 -- 15  [label=3];
+  14 -- 40  [label=6];
+  15 -- 16  [label=3];
+  15 -- 18  [label=3];
+  16 -- 17  [label=3];
+  16 -- 19  [label=3];
+  17 -- 18  [label=3];
+  18 -- 19  [label=3];
+  19 -- 20  [label=3];
+  20 -- 21  [label=3];
+  20 -- 29  [label=3];
+  21 -- 24  [label=3];
+  21 -- 22  [label=3];
+  21 -- 41  [label=3];
+  22 -- 23  [label=3];
+  23 -- 24  [label=3];
+  25 -- 26  [label=3];
+  26 -- 27  [label=3];
+  26 -- 32  [label=3];
+  27 -- 28  [label=3];
+  27 -- 33  [label=3];
+  28 -- 29  [label=3];
+  29 -- 30  [label=3];
+  30 -- 31  [label=3];
+  31 -- 39  [label=3];
+  32 -- 34  [label=6];
+  33 -- 34  [label=6];
+  34 -- 35  [label=6];
+  34 -- 36  [label=6];
+  36 -- 37  [label=3];
+  37 -- 38  [label=3];
+  38 -- 39  [label=3];
+  39 -- 40  [label=3]; /* ???? */
+  42 -- 43  [label=3];
```

Comment:
Missing 40 -- 41. Is this okay?

Reply:
Yeah, the communication model was pregenerated using the meshes in https://github.com/osrf/subt/tree/master/subt_ign/worlds/final_event/08/comms_tiles